clients/ts/signalr/src/LongPollingTransport.ts

```diff
@@ -22,6 +23,12 @@ export class LongPollingTransport implements ITransport {
     private shutdownTimer: any; // We use 'any' because this is an object in NodeJS. But it still gets passed to clearTimeout, so it doesn't really matter
     private shutdownTimeout: number;
     private running: boolean;
+    private stopped: boolean;
+
+    // This is an internal type, not exported from 'index' so this is really just internal.
+    public get pollAborted() {
```

Comment:
getPollAborted?

Reply:
See above comment. This is a property getter definition